roles/openshift_monitor_availability/tasks/install_web_console_smoke_test.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Who creates the user the app uses for console login?

Reply:
It uses the service account token.

Reply:
@ironcladlou those vars linger from local dev, for using OAuth.  I don't know if we would ever want to validate that flow in production, so we can prob remove.